flow360/component/simulation/meshing_param/params.py

```diff
@@ -18,23 +19,58 @@ class MeshingParameters(Flow360BaseModel):
     1. Add rotational zones.
     2. Add default BETDisk refinement.
 
-    Attributes:
-    ----------
-    farfield: Optional[Farfield]
-        Farfield type for meshing.
-    refinement_factor: Optional[pd.PositiveFloat]
-        If refinementFactor=r is provided all spacings in refinement regions and first layer thickness will be adjusted to generate r-times finer mesh. For example, if refinementFactor=2, all spacings will be divided by 2**(1/3), so the resulting mesh will have approximately 2 times more nodes.
-    gap_treatment_strength: Optional[float]
-        Narrow gap treatment strength used when two surfaces are in close proximity. Use a value between 0 and 1, where 0 is no treatment and 1 is the most conservative treatment. This parameter has a global impact where the anisotropic transition into the isotropic mesh. However, the impact on regions without close proximity is negligible.
-    refinements: Optional[List[Union[EdgeRefinementTypes, FaceRefinement, ZoneRefinementTypes]]]
-        Refinements for meshing.
+    Affects volume meshing:
+    - farfield
+    - refinement_factor
+    - gap_treatment_strength
+    - volume_layer_growth_rate
+    - volume_layer_first_layer_thickness
+    - `class` UniformRefinement
+    - `class` CylindricalRefinement
+
+    Affects surface meshing:
+    - max_edge_length
+    - curvature_resolution_angle
+    - surface_layer_growth_rate
+    - `class` FaceRefinement
+    - `class` AnisoSurfaceEdge
+    - `class` ProjectAnisoSurfaceEdge
     """
 
-    # Global fields:
-    farfield: Optional[Farfield] = pd.Field()
-    refinement_factor: Optional[pd.PositiveFloat] = pd.Field()
-    gap_treatment_strength: Optional[float] = pd.Field(ge=0, le=1)
+    # Volume **defaults**:
+    farfield: Literal["auto", "quasi-3d", "user-defined"] = pd.Field(
+        description="Type of farfield generation."
+    )
+    refinement_factor: pd.PositiveFloat = pd.Field(
+        description="If refinementFactor=r is provided all spacings in refinement regions and first layer thickness will be adjusted to generate r-times finer mesh."
+    )
+    gap_treatment_strength: float = pd.Field(
+        ge=0,
+        le=1,
+        description="Narrow gap treatment strength used when two surfaces are in close proximity. Use a value between 0 and 1, where 0 is no treatment and 1 is the most conservative treatment. This parameter has a global impact where the anisotropic transition into the isotropic mesh. However, the impact on regions without close proximity is negligible.",
+    )
+    volume_layer_growth_rate: pd.PositiveFloat = pd.Field(
+        gt=1, description="Growth rate for volume prism layers."
+    )
+    volume_layer_first_layer_thickness: pd.PositiveFloat = pd.Field(
+        description="Default first layer thickness for volumetric anisotropic layers."
+    )
```

Comment:
I think this is not needed. It can be achieved by:
```python
MeshingParameters(refinements=[BoundaryLayerRefinement(entities="*", first_layer_thickness=...,  growth_rate=...)]
```

Reply:
Yes, the issue is that:
1. We cannot just assume a value for default. Will the user be happy if he forget one face and that found that face grow with some strange first layer thickness for example?
2. Then We need to make sure the user have this line of code to provide the base line which is not intuitive. How is the user supposed to know he should always begin with:
```python
MeshingParameters(refinements=[BoundaryLayerRefinement(entities="*", first_layer_thickness=...,  growth_rate=...)]
```

Reply:
As discussed, we omit "entities" to specify global properties.